app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list:
+    new_list = [Person(people[i]["name"], people[i]["age"])
+                for i in range(len(people))]
+    for obj in range(len(new_list)):
+        if people[obj].get("wife"):
+            new_list[obj].wife = Person.people[people[obj].get("wife")]
+
+        if people[obj].get("husband"):
+            new_list[obj].husband = Person.people[people[obj].get("husband")]
```

Comment:
Similar to the `wife` key, the code assumes that the `husband` key will always correspond to an existing `Person` in `Person.people`. If the `husband` does not exist, this will raise a `KeyError`. Consider adding a check to ensure the `husband` exists before assigning.